src/Common/Commands.Common.Storage/Commands.Common.Storage.csproj

```diff
@@ -50,8 +50,90 @@
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Net" />
+    <Reference Include="System.Runtime.Serialization" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="Microsoft.Rest.ClientRuntime">
+      <HintPath>..\..\packages\Microsoft.Rest.ClientRuntime.2.3.11\lib\net452\Microsoft.Rest.ClientRuntime.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime.Azure">
+      <HintPath>..\..\packages\Microsoft.Rest.ClientRuntime.Azure.3.3.12\lib\net452\Microsoft.Rest.ClientRuntime.Azure.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Newtonsoft.Json">
+      <HintPath>..\..\packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="Generated.Version2017-10-01\IOperations.cs" />
```

Comment:
@praries880 currently the `Commands.Common.Storage` project contains adapters around different versions of the Storage data plane SDK (`WindowsAzure.Storage`), so I'm not sure if we want to put generated _management_ SDK code into this project. We should talk to Mark about what the right location for this code is (this project or a separate project)

Reply:
@praries880 talked to Mark -- we should create a new project `Commands.Common.Storage.Management` and throw this generated code into that new project

Reply:
will do